bootstrap/src/lib/cache-warmer/actions.ts

```diff
@@ -0,0 +1,48 @@
+import { AdapterRequest, Execute } from '@chainlink/types'
+import { createAction } from '@reduxjs/toolkit'
+
+interface WarmupRequestSubscribedPayload extends AdapterRequest {
+  executeFn: Execute
```

Comment:
Yup, in the ideal world i would have used something like https://github.com/microsoft/tsyringe to inject deps. This is a hack for two reasons:

1. It lets me set the `executeFn` every single time a new external request comes in, to mitigate https://github.com/smartcontractkit/external-adapters-js/blob/develop/bootstrap/src/index.ts#L69
2. Easier to do rather than re-jig the entire app to support dep injection since we're moving off of this codebase in the near feature